Modules/_winapi.c

```diff
@@ -2160,6 +2205,32 @@ static int winapi_exec(PyObject *m)
     WINAPI_CONSTANT(F_DWORD, FILE_TYPE_PIPE);
     WINAPI_CONSTANT(F_DWORD, FILE_TYPE_REMOTE);
 
+    WINAPI_CONSTANT("u", LOCALE_NAME_INVARIANT);
+    WINAPI_CONSTANT(F_DWORD, LOCALE_NAME_MAX_LENGTH);
+    WINAPI_CONSTANT("u", LOCALE_NAME_SYSTEM_DEFAULT);
+    WINAPI_CONSTANT("u", LOCALE_NAME_USER_DEFAULT);
+
+    WINAPI_CONSTANT(F_DWORD, LCMAP_FULLWIDTH);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_HALFWIDTH);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_HIRAGANA);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_KATAKANA);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_LINGUISTIC_CASING);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_LOWERCASE);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_SIMPLIFIED_CHINESE);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_TITLECASE);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_TRADITIONAL_CHINESE);
+    WINAPI_CONSTANT(F_DWORD, LCMAP_UPPERCASE);
+    WINAPI_CONSTANT(F_DWORD, NORM_IGNORENONSPACE);
+    WINAPI_CONSTANT(F_DWORD, NORM_IGNORESYMBOLS);
+    WINAPI_CONSTANT(F_DWORD, LINGUISTIC_IGNORECASE);
+    WINAPI_CONSTANT(F_DWORD, LINGUISTIC_IGNOREDIACRITIC);
+    WINAPI_CONSTANT(F_DWORD, NORM_IGNORECASE);
+    WINAPI_CONSTANT(F_DWORD, NORM_IGNOREKANATYPE);
+    WINAPI_CONSTANT(F_DWORD, NORM_IGNOREWIDTH);
+    WINAPI_CONSTANT(F_DWORD, NORM_LINGUISTIC_CASING);
+    WINAPI_CONSTANT(F_DWORD, SORT_DIGITSASNUMBERS);
+    WINAPI_CONSTANT(F_DWORD, SORT_STRINGSORT);
```

Comment:
`LCMAP_SORTKEY` isn't supported, so skip defining these constants. However, there's no need to explicitly fail if they're passed in `flags`.